### PR TITLE
LPS-186292 | Test Fix | Master

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-186292

cc: @patriciajeanperez

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/usecase/DynamicdatalistsUsecase.testcase [hunk 1]
```diff
@@ -2350,57 +2350,62 @@ definition {
...
+				ddlListName = "List Name");
+
+			DDLRecord.populateFieldLinkToPage(
+				ddlRecordFieldData = "Test Page 10",
+				fieldName = "LinkToPage");
 
-		DDLRecord.populateFieldLinkToPage(
-			ddlRecordFieldData = "Test Page 10",
-			fieldName = "LinkToPage");
+			SelectFrameTop();
```
Comment (kristinonias) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/usecase/DynamicdatalistsUsecase.testcase:2397]: Added this to ensure that publish was being clicked

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/usecase/DynamicdatalistsUsecase.testcase [hunk 2]
```diff
@@ -2350,57 +2350,62 @@ definition {
...
 
-		Click(
-			key_indexNumber = 1,
-			key_text = "Publish",
-			locator1 = "Button#ANY_N");
+			Button.clickPublish();
```
Comment (kristinonias) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/usecase/DynamicdatalistsUsecase.testcase:2399]: Changed the click to the macro to be cleaner.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/usecase/DynamicdatalistsUsecase.testcase [hunk 3]
```diff
@@ -2350,57 +2350,62 @@ definition {
 				layoutName = "Test Page ${pageNumber}");
 		}
 
-		DDLList.openDynamicDataListsAdmin(siteURLKey = "guest");
+		task ("Given multiple pages with a field linked") {
```
Comment (kristinonias) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/dynamicdatalists/usecase/DynamicdatalistsUsecase.testcase:2353]: Added task blocks

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
Hey @kristinonias, can you upload the passing html file?

**Comment by kristinonias** [comment]
> Hey @kristinonias, can you upload the passing html file?

It should be on there now!

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by patriciajeanperez** [comment]
ci:forward:force

**Pull request closed without merging**